modules/virt-configuring-certificate-rotation.adoc

```diff
@@ -0,0 +1,42 @@
+
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-configuring-certificate-rotation.adoc
+
+[id="virt-configuring-certificate-rotation_{context}"]
+
+Configure certificate rotation parameters to replace existing certificates. You can do this during OpenShift Virtualization installation in the web console or after installation in the YAML file.
+
+.Procedure
+
+. Open the `HyperConverged` Custom Resource (CR) by running the following command:
```

Comment:
If you take my suggestion above, you can just call it the `HyperConverged` CR here. Also note that "custom resource" shouldn't be capitalized (more info: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#api-object-formatting).